hw/top_earlgrey/data/ip/chip_aon_timer_testplan.hjson

```diff
@@ -0,0 +1,125 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_aon_timer
+  testpoints: [
+    {
+      name: chip_sw_aon_timer_wakeup_irq
+      desc: '''Verify the AON timer wake up interrupt in normal operating state.
+
+            - Program the PLIC to let the AON timer wake up interrupt the CPU.
+            - Program the AON timer to generate the wake up timeout interrupt after some time.
+            - Issue a WFI to wait for the interrupt to trigger.
+            - Service the interrupt when it triggers; verify that it came from AON timer.
+            - Verify that the interrupt triggered only after the timeout elapsed.
+            '''
+      stage: V2
+      si_stage: SV2
+      features: ["AON_TIMER.INTERRUPT.WAKEUP"]
+      tests: ["chip_sw_aon_timer_irq"]
```

Comment:
Add bazel test for fpga below if it exists and runs. In this case it does, so add
    bazel: ["//sw/device/tests:aon_timer_irq_test_fpga_cw310_test_rom"]
This is not an asic bazel test, but mentioning this target makes it known the effort is very small to change this to run in the asic.

Reply:
Done